vm-zone-move/README.md

```diff
@@ -0,0 +1,48 @@
+# Failover scripts
+
+During zone down scenario, the VM cannot be accessed. These scripts will bring back the vm in a different zone or region depending upon the type of disks (ZRS/LRS) attached to the vm. Affectively moving the VM. 
+
+## How to Use? 
+
+* Ensure that python is installed on the machine where these scripts are going to be run. 
+* Install the required python packages as mentioned in requirements.txt
+* Run main.py with the arguments as the details of the affected VM in the zonal outage. In case all the requirements are met, the VM will be brought back up in a different zone or region with the disks attached. 
```

Comment:
Better to illustrate with example commands

Reply:
added example command in readme